### ISSUE TITLE
Interrogans a principi de pregunta

### ISSUE THREAD

**Issue by jordimas** [apertium/apertium-cat#16]
Quan traduïm del català al castellà:

Teniu concedida alguna beca o ajut del curs 2020-21? > Tenéis concedida alguna beca o ayuda del curso 2020-21?

Hauríem de dir "¿Tenéis concedida alguna beca o ayuda del curso 2020-21?"

En castellà cal afegir-hi l'interrogant a principi de frase interrogativa.

**Comment by jaumeortola** [comment]
Sí. Pot haver-hi una petita dificultat per a saber on cal posar l'interrogant inicial. En la majoria de casos serà al principi de la frase.

No tinc ni idea de com es pot fer això en Apertium. Podria ser en postedició? @MarcRiera No hi ha res que correspongui a divisions de frase o marques de principi i final de frase.

**Comment by MarcRiera** [comment]
L'altre dia m'ho vaig rumiar i es pot fer amb Constraint Grammar. Seria afegir un pas entre el trànsfer i la generació que afegís els signes d'interrogació i exclamació d'obertura. Fins aquí, bé.

El problema és que posar aquests signes no és una tasca fàcil perquè depèn molt del context i seria un enfocament bastant superficial. Quan no hi ha comes, és fàcil. Quan hi ha comes pel mig, es complica i no podem encertar tots els casos. De totes maneres, a parer meu, si podem encertar una part dels casos ja és una millora en comparació amb el que tenim ara.
